aws-android-sdk-core/src/main/java/com/amazonaws/internal/keyvaluestore/AWSKeyValueStore.java

```diff
@@ -471,11 +464,11 @@ private AlgorithmParameterSpec getInitializationVector(final String keyOfDataInS
     }
 
     private AlgorithmParameterSpec getAlgorithmParameterSpecForIV(byte[] iv) {
-        return
-            //@apiLevel23Start
-            apiLevel >= ANDROID_API_LEVEL_23 ? new GCMParameterSpec(CIPHER_AES_GCM_NOPADDING_TAG_LENGTH_LENGTH_IN_BITS, iv) :
-            //@apiLevel23End
-            new IvParameterSpec(iv);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Android Lint can't figure out that this code is correctly guarded, unless these standard `Build` values are used.